render.c

```diff
@@ -108,6 +109,13 @@ render_create(struct teletext_struct* p_teletext,
   width = (640 + (border_chars * 2 * 16));
   height = (512 + (border_chars * 2 * 16));
 
+  if (util_has_option(
+      p_options->p_opt_flags, "video:double-size")) {
+    width *= 2;
+    height *= 2;
+    p_render->is_double_size = 1;
+  } else p_render->is_double_size = 0;
```

Comment:
Style: beebjit always does
if () {
} else {
}

even if the else statement is a single statement.